### ISSUE TITLE
FragmentPagerAdapter   出现的问题1

### ISSUE THREAD

**Issue by huangmaohuoshaoxuexia** [huangmaohuoshaoxuexia/Note#54]
问题描述： 有4个或者多个fragment 时  点击后两个然后再切换到第一个fragment ,里面的数据都是重复显示 最后一行的数据
例如：名称：111111111111111
          类型：111111111111111
          备注：111111111111111（正确的字段）
原因：FragmentPagerAdapter  中少些 destroyItem 方法
解释：destroyItem()该函数被调用后，会对 Fragment 进行 FragmentTransaction.detach()。这里不是 
remove()，只是 detach()，因此 Fragment 还在 FragmentManager 管理中，Fragment 所占用的资源不会被释放。与remove()不同的是，所有绑定的事件、附加的数据等都会保留下来
尝试 1：加入 pager.setOffscreenPageLimit(2); 就可以让ViewPager多缓存一个页面（请求三次）
              效果：点击第三个  然后回到第一个时正常  ，但是点击第四个  然后回到第一个时就不正常 
（谢谢 何明阳的帮助）
